index.js

```diff
@@ -221,6 +221,12 @@ function grype_render_rules(vulnerabilities) {
         let ruleID = "ANCHOREVULN_"+v.vulnerability.id+"_"+v.artifact.type+"_"+v.artifact.name+"_"+v.artifact.version;
         if (!ruleIDs.includes(ruleID)) {
           ruleIDs.push(ruleID);
+          // Entirely possible to not have any links whatsoever
+          var link = v.vulnerability.id;
+          if ("links" in v.vulnerability) {
```

Comment:
I haven't seen this style of check before, but that doesn't mean it's wrong. I usually see checks for property's existence in the form of `if (typeof v.vulnerability.links !== "undefined") ...`